src/Credentials/CredentialProvider.php

```diff
@@ -872,11 +891,11 @@ private static function getSsoCredentials($profiles, $ssoProfileName, $filename,
             $ssoSession = $profiles['sso-session ' . $ssoProfile['sso_session']];
             $ssoOidcClient = new Aws\SSOOIDC\SSOOIDCClient([
                 'region' => $ssoSession['sso_region'],
-                'version' => '2019-06-10',
+                'version' => 'latest',
```

Comment:
Why has this changed?

Reply:
Sorry, I didn't mean to commit that change. You can savely revert it. Thx.

Reply:
This line is no longer significant and can now be safely deleted; the `version` input parameter is a consequence of the fact that this SDK is one of the oldest ones we have and the vision for service versioning was very different at the time that this was written.  Services now rarely come out with new versions, and when they do we require those versions to be backward compatible with previous versions.  

Every other SDK defaults to `latest`; we recently made a change so that customers no longer have to specify the version when initializing a client and it will always default to latest.